packages/config/config/devices/0x027a/templates/zooz_template.json

```diff
@@ -252,6 +276,88 @@
 			}
 		]
 	},
+	"external_switch_type_0-3": {
+		"label": "External Switch Type",
+		"valueSize": 1,
+		"unsigned": true,
+		"minValue": 0,
+		"maxValue": 3,
+		"defaultValue": 2,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Toggle switch",
+				"value": 0
+			},
+			{
+				"label": "Momentary switch",
+				"value": 1
+			},
+			{
+				"label": "Toggle switch with fixed actions",
+				"value": 2
+			},
+			{
+				"label": "3-way impulse control",
```

Comment:
What does 3-way impulse control mean?

Reply:
These are pretty new devices and the documentation from Zooz is still a little sparse. According to the [advanced settings article here](https://www.support.getzooz.com/kb/article/796-zen51-dry-contact-relay-advanced-settings/):

>(for selected 3-way scenarios, refer to the notes in the wiring instructions for your set-up)

However, the manuals don't explain this; instead they have QR codes that link back to the knowledge base article above. A circular reference!

I reckon, based on the features of other relays like the PAN03 or the Aeotec Nanos, that this setting makes the relay look for a short current pulse from a circuit like the ones depicted here: https://help.aeotec.com/support/solutions/articles/6000176583-nano-dimmer-wiring-3-way-4-way-5-way-switch-methods-

These are for scenarios where the line is in the wrong gang box for a standard 3-way setup. Unfortunately I haven't tested these cases, so I can't be sure that Zooz means the same thing that Aeotec means with their relays. I could open a ticket with them, I suppose, but I don't think I'd get around to opening up my wall and testing it.

In the meantime, should I just add Zooz's frustraing blurb about referring to other wiring instructions that don't exist yet? Or should I paraphrase what I said above, in as few words as possible?

Reply:
I'd just play the ball back to Zooz and refer to the device manual / wiring instructions. We should probably play nice and tell them that they are nowhere to be found.